examples/kitty/src/ethernet_config/ethernet_config.h

```diff
@@ -2,9 +2,8 @@
 
 #include <sddf/network/queue.h>
 
-#define NUM_CLIENTS 3
+#define NUM_CLIENTS 2
```

Comment:
This should probably be renamed to `NETWORK_NUM_CLIENTS` for the latest sDDF change I think.